apps/discovery-platform/src/constants.ts

```diff
@@ -0,0 +1,44 @@
+const {
+  // Api endpoint used for completing funding requests of registered nodes
+  FUNDING_SERVICE_URL,
+  // Access token used to connect to hoprd entry node
+  HOPRD_ACCESS_TOKEN,
+  // Database connection url
+  DB_CONNECTION_URL,
+} = process.env;
+
+// Skips commitment check making all fresh nodes go to ready
+const SKIP_CHECK_COMMITMENT =
+  Boolean(process.env.SKIP_CHECK_COMMITMENT) ?? false;
```

Comment:
This can be error-prone, as we are expecting a string, u only need to do `process.env.SKIP_CHECK_COMMITMENT === "true"`